### DISCUSSION TITLE
Comments on two commits

### DISCUSSION THREAD

**Discussion by tmob03** [ArgosOfIthica/scalu#39] [General]
I left comments on two recent commits, but I'm making this thread just as a central place to discuss both.

The changes I commented on result in resulting scripts using the std module to be much bigger in file size. This, IMO, should be avoided at all costs. Filesize should be as small as possible so the std module is not taking up (much) of the filesize limit per cfg file. I know this can be gotten around with multiple files, but this makes it much more of a pain to distribute.

**Comment by ArgosOfIthaca** [comment]
For https://github.com/ArgosOfIthica/scalu/commit/1c52c4e1de070b10f830d494017f592bab7e8f0d:

I agree, and that's why the branch hasn't been merged yet. Having a std sandbox is just a temporal step towards having an actually good std macro set, which is what I'm currently working on. I would like the work you've done to be more convenient to use, i.e., simply emit a print macro when debugging rather than go through the ritual of breaking every sandbox convention just to know the current value of a number. This would also take care of all of our file size bloat problems.

For https://github.com/ArgosOfIthica/scalu/commit/07195917703844ac5518ad281604f98f59ce908a:

I actually agree that this sort of thing is a need, but rather than provide ascii subsets for every use case, we're going to end up providing a special macro that generates exactly the unicode subset that you desire.
